### PR TITLE
Initial build 0.0.7

### PR DESCRIPTION
**Pull request by ViridianMelody**
Jira: https://anaconda.atlassian.net/browse/PKG-2053
Upstream: https://github.com/fastai/fastdownload

The commit history is a bit messy because I accidentally pushed my changes to main, but all of the changes are listed here.

`fastai` <- `fastdownload`

- Added `packaging` to test requires because it was causing pip check to fail due to a discrepancy in the `fastcore` recipe.
- Minor adjustments to conda-forge.

### CODE REVIEW

**Review by ELundby45 — APPROVED** [review]

File: abs.yaml
Comment (marcoesters) [inline abs.yaml]: Please remove

File: recipe/meta.yaml
Comment (marcoesters) [inline recipe/meta.yaml]: That's because `fastcore` is missing it. See here for the requirements: https://github.com/fastai/fastcore/blob/master/setup.py#L29

And here is the recipe: https://github.com/fastai/fastcore/blob/1.5.29/setup.py#L29

When a `pip check` fails, the reason is often a broken dependency. It looks like `pip check` in fastcore didn't catch that.

`fastcore` should be rebuilt or hotfixed first before merging this PR.

**Review by marcoesters — COMMENTED** [review]
I'm not requesting changes here because I will be OOO soon and don't want to block it,

However, a dependency needs to be fixed first before merging this PR.

File: recipe/meta.yaml
Comment (ELundby45) [reply recipe/meta.yaml]: See here a discussion why packaging was left out intentionally as requested by reviewers.
https://github.com/AnacondaRecipes/fastcore-feedstock/pull/2

This should not block this PR and can be fixed by a patch in `fastcore` when it can/needs to be rebuilt

Comment (ELundby45) [reply recipe/meta.yaml]: Here's a patched `fastcore` for review https://github.com/AnacondaRecipes/fastcore-feedstock/pull/3/files

Comment (marcoesters) [reply recipe/meta.yaml]: Thanks for the clarification! A comment would be really helpful here, too, about why `pip check` fails

**Review by marcoesters — APPROVED** [review]
This looks good, but I strongly encourage to adjust the comment to explain why `pip check` fails here.

**Review by skupr-anaconda — APPROVED** [review]
LGTM

**Pull request merged**
